go/scheduler/api/api.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"strings"
 
+	tmtypes "github.com/tendermint/tendermint/types"
```

Comment:
Ow, this brings in tendermint to the consensus agnostic part (the scheduler API). I am fine with defining the constant such that it satisfies the Tendermint limits and mention that the constant is such that it works with the Tendermint backend, but we should not import tendermint here. In case this ever changes in Tendermint, the tendermint-specific sanity check (that imports tendermint) will fail.

Reply:
ok